src/components/LandingPage.vue

```diff
@@ -120,6 +120,20 @@
                     arrow_downward
                   </span>
                 </div>
+                <div
+                  v-if="addonSelected == 'Plater'"
```

Comment:
I'd prefer this to be not hard-coded but possibly tied to a check if values exist in the column.
The reason for this lies in the modular approach to adding other supported addons which might use this column as well.
Possibly add a flag in the `addonConfigs` and check for the current selected addon tab?

Reply:
I get it, it didn't feel right to me either. I can either check if one of the value in the column has the property auraTypeDisplay, that'd be the easiest.
But like you said, I can use the `addonConfigs`.I can either use a flag, or an array with the possibles column values, in case we add some, or want more modularity.

Reply:
Thank you. Looks good to me.